lib/tools/settings-client-commands.js

```diff
@@ -442,4 +444,27 @@ commands.getSmsList = async function getSmsList (opts = {}) {
   return parseJsonData(output, 'SMS list');
 };
 
+/**
+ * Types the given Unicode string.
+ * It is expected that the focus is already put
+ * to the destination input field before this method is called.
+ *
+ * @param {string} text The string to type
+ * @returns {boolean} `true` if the input text has been successfully sent to adb
+ */
+commands.typeUnicode = async function typeUnicode (text) {
+  if (_.isNil(text)) {
```

Comment:
Why this check and not just having a default string of `''` and allowing the check below to return `false`?

Reply:
I want all values except of null and undefined to be actually printed as they are (false -> `false`, `0` -> '0', etc). The `text` parameter will be received from the client via `mobile:` command

Reply:
Ok. Only `null` would work differently with a default parameter.

Reply:
yep, I know. Thanks for the hint